plio/tests.py

```diff
@@ -0,0 +1,111 @@
+from rest_framework.test import APIClient
+from rest_framework.test import APITestCase
+from rest_framework import status
+from oauth2_provider.models import Application
+from django.urls import reverse
+from plio.settings import API_APPLICATION_NAME, OAUTH2_PROVIDER
+import datetime
+import random
+import string
+from users.models import User
+from oauth2_provider.models import AccessToken
+from plio.models import Plio, Video
```

Comment:
clubbing together imports and following PEP8's guidelines for order of imports: pre-installed packages, installed packages and then, current package.
```suggestion
import datetime
import random
import string

from rest_framework.test import APIClient
from rest_framework.test import APITestCase
from rest_framework import status
from oauth2_provider.models import Application
from oauth2_provider.models import AccessToken
from django.urls import reverse

from users.models import User
from plio.settings import API_APPLICATION_NAME, OAUTH2_PROVIDER
from plio.models import Plio, Video
```

Reply:
cool. thanks for the description also.